conda_build/main_build.py

```diff
@@ -255,6 +255,7 @@ def execute(args, parser):
     import shutil
     import tarfile
     import tempfile
+    from os import makedirs
```

Comment:
You mean the fact that it's inside the function? There's probably not a good reason for it.